src/context/walletContext/zeroDevContext.context.tsx

```
@@ -0,0 +1,330 @@
+
+    // TODO: better docstrings
+    // used when data is already encoded from Peanut
+    // but remains unsigned
+    const handleSendUserOpEncoded = async (
+        {
+            to,
+            value,
+            data
+        }: {
+            to: string,
+            value: BigInt,
```

Biome

[error] Don't use 'BigInt' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'bigint' instead


(lint/complexity/noBannedTypes)